app/src/hbs_templates/main.hbs

```diff
@@ -12,173 +12,240 @@
 
 {{>progress_indicator}}
 
-<div class="slides-container">
-  <div class="slide" id="slide-1">
-
-    <div class="mobile">
-      {{! LanguageToggle mobile / tablet only }}
-      {{>language_toggle}}
-    </div>
-
-    <div class="centered">
-      <img class="triple-s" src="assets/png/triple-sss.png" alt="">
-      <img class="building intro" src="assets/png/building.png" alt="">
-      <h1 class="font-style-hero landing">{{ slide01.h1 }}</h1>
-      <p class="font-style-secondary sub-head">{{ slide01.subhead }}</p>
-      <p class="landing">{{ slide01.landing }}</p>
-      <p class="go-step4">
-        <a class="go-step4" href="#slide-7">
-          {{ slide01.go_step_4 }}
-        </a>
-      </p>
-    </div>
-    <button class="go-next bottom-arrow">
-      <div class="button-text">{{nav_bottom_arrow.begin}}</div>
-      <div class="arrow"></div>
-    </button>
-  </div>
-
-  <div class="slide" id="slide-2">
-    <div class="centered">
-      <h2 class="font-style-hero step">{{slide02.h2}}</h2>
-      <p class="step">{{slide02.privacy}}</p>
-      {{>address_search_form}}
-    </div>
-  </div>
-
-  <div class="slide" id="slide-3">
-    <div class="centered v-centered">
-      <h2 class="font-style-hero checking"> {{slide03.checking }} </h2>
-      <p> {{slide03.p}} </p>
-      <div id="sprite"></div>
-    </div>
-  </div>
-
-  <div class="slide" id="slide-4">
-    <div class="centered">
-      <div class="yes hidden">
-        <h2 class="font-style-hero">{{{slide04.yes.h2}}}</h2>
-        <p>{{{slide04.yes.p}}}</p>
+<section
+  class="slides-container"
+  aria-label="A step by step guide to Rent Stabilization in New York City"
```

Comment:
add a `TODO` code comment for the need of a translation string, here and for the other `aria-label` and `aria-roledescription` values.